traffic_monitor/cache/stats_over_http.go

```diff
@@ -19,17 +19,19 @@ package cache
  * under the License.
  */
 
-import "errors"
-import "fmt"
-import "io"
-import "math"
-import "strings"
-import "strconv"
+import (
+	"errors"
+	"fmt"
+	"io"
+	"math"
+	"strconv"
+	"strings"
 
-import "github.com/apache/trafficcontrol/lib/go-log"
-import "github.com/apache/trafficcontrol/traffic_monitor/todata"
+	"github.com/apache/trafficcontrol/lib/go-log"
 
-import "github.com/json-iterator/go"
+	"github.com/apache/trafficcontrol/traffic_monitor/todata"
+	jsoniter "github.com/json-iterator/go"
+)
```

Comment:
These imports should be split into (at least) 3 groups:
- core package imports
- ATC imports
- external library imports